README.md

```diff
@@ -12,14 +12,14 @@ To use this package, add `ffi` and `tizen_interop` as dependencies in your `pubs
 ```yaml
 dependencies:
   ffi: ^2.0.1
-  tizen_interop: ^0.2.7
+  tizen_interop: ^0.3.0
```

Comment:
Please update the `pubspec.yaml` file.

Reply:
thank you for your check. i updated it

Reply:
Also make sure to publish the package on pub.dev after merge. It should be done manually by triggering the [release](https://github.com/flutter-tizen/tizen_interop/actions/workflows/release.yml) workflow (`workflow_dispatch`). I created a release for the 0.2.7 version.